main.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+class Todo {
+  constructor() {
+    this.list = [];
+    this.id = 1;
+  }
+
+  addItem(title, priority) {
+    const item = {
+      id: this.id,
+      title: title,
+      priority: priority
+    };
+
+    this.id++;
+    this.list.push(item);
+
+    return item.id;
+  }
+
+  removeItem(id) {
+    const index = this.list.findIndex((item, i)=> {
+      if (item.id === id) {
+        return i;
+      }
+    });
+    if (!index) {
```

Comment:
What if I need to remove the first element or findIndex return `-1` https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex